README.md

```diff
@@ -319,7 +319,7 @@ release:
 ```
 
 ```bash
-earth --secret GITHUB_TOKEN --push +release
+earthly --secret GITHUB_TOKEN --push +release
```

Comment:
Make changes in README on `next` branch instead?

Reply:
that's a good call, I left out the docs renaming for a different PR but let the README changes slip in, i'll save these for a future PR.